packages/synapse-react-client/src/utils/AppUtils/session/ApplicationSessionManager.tsx

```diff
@@ -101,8 +101,8 @@ export function ApplicationSessionManager(
       )
       const now = dayjs.utc()
       if (now.diff(lastAuthenticatedOn, 'seconds') > maxAge) {
-        // Don't set the token so the user must re-authenticate to use this app
-        setHasInitializedSession(true)
+        // Invalidate the token (if present) so the user must re-authenticate to use this app
+        initAnonymousUserState()
```

Comment:
explicitly signout the user (invalidate the token), set the token to undefined, and set hasInitializedSession to true (all done in initAnonymousUserState())